addons/mail/models/mail_template.py

```diff
@@ -32,7 +32,9 @@ def default_get(self, fields):
     model_id = fields.Many2one('ir.model', 'Applies to', help="The type of document this template can be used with")
     model = fields.Char('Related Document Model', related='model_id.model', index=True, store=True, readonly=True)
     subject = fields.Char('Subject', translate=True, help="Subject (placeholders may be used here)")
-    email_from = fields.Char('From',
+    author = fields.Char('From (User)',
+        help="Author of the mail. If set and not email_from, we will use the email of the user specified.")
```

Comment:
Actually why not return `create_uid` (or it's corresponding partner) instead of adding extra fields? I'm not sure to understand the point of this, the name of this field is probably not ideal and the help text doesn't really help.

Reply:
We don't want to set a "default" value for this. For example, like the email_from, we can use `{{object.user_id.id}}` to have a dynamic value for the author.